### PR TITLE
Beta 2023.05.24

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/Ciel/Day 10/10_01_CIEL10_9.txt [hunk 1]
```diff
@@ -561,7 +561,7 @@ Not even Ciel found out about this place, so how on earth did this woman know?
 [sha:f74d8fce2a19cc639c284721a754632e8f18f4dd]{
 -- Page 635, Offset 28878.
 -- だってあれ強者が弱者になっちゃう話じゃない？　愛が足りないんだってば、愛が」
-I mean, isn't it just the story someone strong becoming a total weakling?
+I mean, isn't the story just someone strong becoming a total weakling?
```
Comment (yk1512) [inline script/Ciel/Day 10/10_01_CIEL10_9.txt:564]: Actually, I think I see the original intent now, was just missing a word. 

Isn't it just the story 'of' someone strong...

**Review by yk1512 — APPROVED** [review]

File: script/Ciel/Day 10/10_01_CIEL10_5_2.txt [hunk 2]
```diff
@@ -503,7 +503,7 @@ No, there's no way it would play out like that.
 [sha:a9cb02d3f16f20cd7e844f89360b58b153423f65]{
 -- Page 408, Offset 28204.
 -- 女は俺を殺さない。
-The woman won't kill me.
+This woman won't kill me.
```
Comment (Petrikow) [inline script/Ciel/Day 10/10_01_CIEL10_5_2.txt:506]: Might as well use 'she'

File: script/Ciel/Day 13/13_01_01_CIEL13_1.txt [hunk 3]
```diff
@@ -820,6 +820,8 @@ If that happened, I would turn into something even more horrifying.
 [sha:8b251ff93fe01a5fc3b929aaa37269e9ecaed252]{
 -- Page 40, Offset 33217.
 -- 「■■■■■。■■■■■■■■■■■、■■■■？」
+// Kinda annoying to fully decipher (can't hear the first part of the first clause)
+// Include TL as ruby?
 "■■■■■. ■■■■■■■■■■■, ■■■■?"
```
Comment (Petrikow) [inline script/Ciel/Day 13/13_01_01_CIEL13_1.txt:825]: >お機嫌はよろしいですか。お辛いでしたら、お声をかけてください。

Is everything in order? If you're feeling down, please feel free to let me know.

Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_1.txt:825]: There's also one earlier at Offset 33182
>志貴さん？　いかがなさいましたか。やはり、お機嫌は？

>Shiki-san? How are you holding up? You're not feeling well, are you?

Comment (Petrikow) [reply script/Ciel/Day 13/13_01_01_CIEL13_1.txt:825]: Maybe we could render these upside down for the trippy effect, or something.
Ruby feels too straightforward

Comment (rschlaikjer) [reply script/Ciel/Day 13/13_01_01_CIEL13_1.txt:825]: Thoughts?


2023-05-31-230251_2160x1286_scrot

**Pull request merged**